helix-view/src/gutter.rs

```diff
@@ -158,7 +165,8 @@ pub fn line_numbers<'doc>(
     let mode = editor.mode;
 
     Box::new(move |line: usize, selected: bool, out: &mut String| {
-        if line == last_line && !draw_last {
+        if line == last_line_in_view && !draw_last {
+            let width = max(n_last_line, n_min);
```

Comment:
Looks like this is computed and used in both branches so it could probably be calculated above with `n_last_line` and `n_min`. You can also use `x.min(y)` and avoid the change to use `std::cmp::max`